core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -852,7 +852,6 @@ private[spark] class TaskSetManager(
           blacklistTracker.foreach(_.updateBlacklistForFetchFailure(
             fetchFailed.bmAddress.host, fetchFailed.bmAddress.executorId))
         }
-
```

Comment:
just for future reference - can you please not change random lines that are not related to the PR?

Reply:
Sure, sorry about that, I think these were left over from my previous commit.